nixos/modules/tasks/network-interfaces.nix

```
@@ -295,6 +295,12 @@ let
           description = "Wether to enable wol on this interface.";
         };
       };
+
+      neededForBoot = mkOption {
+        type = types.bool;
+        default = true;
+        description = "Whether to depend on the presence of this device on boot. Set this to false if this is a pluggable ethernet device (like an Ethernet USB dongle).";
```

Can you break up this line?
```suggestion
        description = ''
          Whether to depend on the presence of this device on boot.
          Set this to <literal>false</literal> if this is a pluggable ethernet device (like an Ethernet USB dongle).
        '';
```